### ISSUE TITLE
anime. looktype isometric tibia-style

### ISSUE THREAD

**Issue by gorzelakradoslaw1994-hue** [bmaltais/kohya_ss#3516]
Hello, I need help. I'm training Lore with Tibia's 4700 Sprite because chatgbt suggests I learn geometry with it. The end result is an isometric character, but it doesn't look like the one from Tibia. The goal is Tibia graphics, but the characters will look like anime. I need help with the settings and how many graphics I should add to replicate the Tibia style. I'll add the desired end result below, along with the graphics I've added to learn Lore. Im use Checkpoint (dreamshaper_8safetensors)

<img width="64" height="64" alt="Image" src="https://github.com/user-attachments/assets/5cc91e80-12e1-43be-a6fe-bd0e8fb40ced" />

<img width="512" height="192" alt="Image" src="https://github.com/user-attachments/assets/3555eb95-0242-40d2-acb6-5b1ffbc1ba66" />

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/29159475-1800-4941-8e56-1f58fa21d0bf" />

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/127e37d1-581c-4f5f-8aff-30a71a171366" />

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/b12959b5-3667-4835-b530-dcb825eaaf3d" />

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/56f73539-1720-4fd6-ab5d-f87d2eac8b3b" />

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/8404ca9e-88bb-4253-a1a8-ecc7ded2b982" />

<img width="512" height="512" alt="Image" src="https://github.com/user-attachments/assets/f6908bd6-64f1-4c29-89f9-e4242ab722a1" />

**Comment by bmaltais** [comment]
> *This was generated by AI during triage.*

## Triage disposition

**Not a software defect** in `kohya_ss` / the GUI. This is a **training methodology / support** request about LoRA style results.

### What the report is asking
- Train a LoRA on Tibia-style isometric sprites (~4700 images)
- Desired look: Tibia / isometric game art
- Observed look: anime-like characters
- Base checkpoint: `dreamshaper_8.safetensors`
- Request: settings guidance and dataset size advice

### Why this is not a bug
kohya_ss configures and launches training; it does not force an “anime” or “Tibia” style. Output style is dominated by:

1. **Base checkpoint bias** — DreamShaper is a general / stylized model; it strongly pulls generations toward anime/illustration looks unless the LoRA and captions overpower that prior.
2. **Dataset + captions** — subject framing, backgrounds, tag quality, and trigger words matter more than raw image count for style lock-in.
3. **Training hyperparameters** — network dim/alpha, LR, steps/epochs, resolution, and whether you train UNet only vs text encoder.

No crash, install failure, or incorrect GUI behavior is described.

### Classification
**Type:** Support / usage question (not Regression, Missing Feature, or Design Flaw)

**Recommended labels:** `question`, `training`, `Troubleshooting`  
**Recommended action:** leave open for community help, or close as “not a bug” if the tracker is for defects only.

### Practical guidance (non-code)
These are training tips only — not a product fix:

- Prefer a **base model closer to the target domain** (pixel / game / low-detail / SD1.5 models trained on similar art) over DreamShaper if the goal is hard Tibia style.
- Use a **consistent trigger token** in captions; caption for style + view (isometric, pixel, limited palette) not only character names.
- Start with a **small clean subset** of best-matching sprites, validate, then scale; 4700 is not required if many images are near-duplicates or mixed styles.
- Keep resolution aligned with source art (often low-res sprites upscaled carefully, or train near native aspect).
- If results stay anime-heavy, lower creative prior: stronger LoRA influence at inference, or a less anime-biased base.

### Out of scope for this tracker as a bug
- Implementing a “Tibia style preset”
- Automatic style correction in the GUI
- Dataset curation on the user’s behalf

No TDD fix plan — there is no failing product behavior to unit-test.

**Comment by bmaltais** [comment]
> *This was generated by AI during triage.*

Closing as not a software defect — this is a training methodology / style-support request, not a bug in kohya_ss. See the triage comment above for guidance.
